service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2,2941 +2,166 @@ openapi: 3.0.3
 info:
   title: Tanagra Service API
   description: github.com/DataBiosphere/tanagra
-  version: 0.0.0
+  version: 0.0.1
 
 servers:
-  - url: /
+- url: /
 
 security:
-  - bearerAuth: []
-  - authorization: [openid, email, profile]
+- bearerAuth: []
+- authorization: [ openid, email, profile ]
 
 paths:
   # --------------- Unauthenticated Paths ---------------
 
-  "/status":
+  /status:
     get:
       security: [ ]
-      summary: Returns the operational status of the service
+      description: Returns the operational status of the service
       operationId: serviceStatus
-      tags: [Unauthenticated]
+      tags: [ Unauthenticated ]
       responses:
         200:
           description: Service can process requests
         500:
-          description: |
-            Service cannot process requests. That might be because dependent services are
-            unavailable, or because there is a problem with the service itself.
+          description: Service cannot process requests. That might be because
+            dependent services are unavailable, or because there is a problem
+            with the service itself
 
-  "/version":
+  /version:
     get:
       security: [ ]
-      summary: Returns the deployed version of the service
+      description: Returns the deployed version of the service
       operationId: serviceVersion
-      tags: [Unauthenticated]
+      tags: [ Unauthenticated ]
       responses:
         200:
           description: Version properties
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/SystemVersion"
-        500:
-          $ref: "#/components/responses/ServerError"
+                $ref: './common.yaml#/components/schemas/SystemVersion'
 
   # --------------- Users ---------------
 
-  "/v2/profile":
-    get:
-      description: Returns the current user's profile information
-      operationId: getMe
-      tags: [Users]
-      responses:
-        200:
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/UserProfile"
-        500:
-          $ref: "#/components/responses/ServerError"
+  /v2/profile:
+    $ref: './users.yaml#/paths/~1getMe'
```

Comment:
Can't say I love these weird refs or having to list all the APIs here. This is the only way to split it?